### ISSUE TITLE
RTNETLINK answers: File exists

### ISSUE THREAD

**Issue by MaikoHermans** [brianhornsby/script.openvpn#52]
Like previously said in issue #31 which was closed without comment. When trying to connect to openvpn through ssh the following error message shows up.

Sat May 21 00:32:43 2016 do_ifconfig, tt->ipv6=0, tt->did_ifconfig_ipv6_setup=0
Sat May 21 00:32:43 2016 /sbin/ip link set dev tun0 up mtu 1500
Sat May 21 00:32:43 2016 /sbin/ip addr add dev tun0 local 192.168.55.6 peer 192.168.55.5
RTNETLINK answers: File exists
Sat May 21 00:32:43 2016 ERROR: Linux route add command failed: external program exited with error status: 2
Sat May 21 00:32:43 2016 Initialization Sequence Completed

I dont seem to be the only one struggling with this. Would love to hear how to fix this.

P.S I understand this has very little to do with your plugin but since i followed the guide you wrote to a par I figured you might be able to figure this out a lot quicker then any of us. So really hope you are able to everyone with this problem out

**Comment by brianhornsby** [comment]
I'm not an OpenVPN expert. I have no idea how to fix that error. The best place to look for an answer is on an OpenVPN forum.

**Comment by MaikoHermans** [comment]
Alright so I messed with it a little more and I came upon the following link:

http://www.faqssys.info/debian-8-jessie-with-openvpn-client/

this is basicly saying that openvpn needs to be called upon differently with debian 8. 
I went on and created a inline .conf file named clientvpn.conf.
after doing so I placed it in the /etc/openvpn directory.

I then ran this command openvpn@clientvpn.service
which ran without any problems and connected my vpn.
So if I'm correct the script you are using needs a change for it to work and the guide isn't up to date anymore.

Could this be correct?
